src/scratchtocatrobat/converter/converter.py

```diff
@@ -1529,7 +1522,8 @@ def _convert_append_to_list_block(self):
     def _convert_insert_at_of_list_block(self):
         [value, position, list_name] = self.arguments
         if position == "last":
-            index_formula = catrobat.create_formula_with_value(self._converted_helper_brick_or_formula_element([list_name], "lineCountOfList:"))
+            index_fe_linecount = self._converted_helper_brick_or_formula_element([list_name], "lineCountOfList:")
+            index_formula = catrobat.create_formula_with_value(self._converted_helper_brick_or_formula_element([index_fe_linecount, 1], "+"))
```

Comment:
I tested it and it does work. But I will change it, as this is a "cleaner" solution.